src/lib/Cashlink.ts

```diff
@@ -103,21 +116,24 @@ class Cashlink {
      * Cashlink balance in luna
      */
     public balance: number | null = null;
+    public state: CashlinkState;
```

Comment:
I know we said we are not going to comment on code style much anymore, but this is a change that really grinds my gears again. There is no reason that the `state` needs to be changed from being indirectly assigned in the constructor arguments to this.

Reply:
Typically every change has a reason ;)
This change enables setting the fee via the `fee` setter in the constructor instead of using `_fee` without that the warning gets triggered.

Reply:
Because `state` needs to be assigned _after_ the `fee` to not trigger the warning. I get it, but it's not nice :(